src/esp/gfx/RenderTarget.h

```diff
@@ -48,7 +48,7 @@ class RenderTarget {
                const Magnum::Vector2& depthUnprojection)
       : RenderTarget{size, depthUnprojection, nullptr} {};
 
-  ~RenderTarget() { LOG(INFO) << "Deconstructing RenderTarget"; }
+  ~RenderTarget() {}
```

Comment:
Any reason this LOG got nuked?

Reply:
We need to rebuild the RenderTarget every time the screen is resized. This spams the terminal output.

Reply:
We don't have a DEBUG level? Fair enough.